subsys/bluetooth/host/Kconfig

```diff
@@ -80,6 +81,9 @@ config BT_HCI_TX_STACK_SIZE
 	  NOTE: This is an advanced setting and should not be changed unless
 	  absolutely necessary
 
+config BT_HCI_TX_STACK_SIZE_WITH_PROMPT
+	bool "Enable prompt for BT_HCI_TX_STACK_SIZE"
```

Comment:
Maybe avoid going into kconfig technicals and rephrase to "Override HCI TX thread stack size default" or something like that?

Reply:
maybe, will wait a day or two in case there are more suggestions and then update.

Reply:
Agree with @joerchan. Sounds like a good prompt string.

Another option is to invert it and make the prompt "Automatically calculate HCI TX thread stack size" or the like.